app/assets/stylesheets/components/_onboarding_message.scss

```diff
@@ -0,0 +1,5 @@
+.onboarding-message {
+  text-align: center;
+  margin: 0;
+  padding: $baseline-unit*6 0;
+}
```

Comment:
Oh yeah, missed that in the markup. Must've been those single quotes... ;)

Totally cool as is.